### PR TITLE
typos

### PR DESCRIPTION
**Pull request by ccoVeille**
- **chore: fix typo in file name**
- **chore: fix wrong comments and names**
- **chore: fix Markdown fenced block identifiers**
- **chore: fix typos**
- **chore: fix invalid aliases for Morse decode**
- **chore: fix acronyms case**

### CODE REVIEW

**Comment by ccoVeille** [comment]
Please find my reviews of your code

**Comment by abhimanyu003** [comment]
Hello @ccoVeille many thanks for your updates.
This looks great, but some test are failing, can you please fix them

**Comment by ccoVeille** [comment]
Oh really? I will check when I would be in front of a computer. I'm on phone now

**Comment by ccoVeille** [comment]
I'm a bit surprised this reported nothing 🤔 

https://github.com/abhimanyu003/sttr/pull/44/checks

**Comment by ccoVeille** [comment]
Could you tellme what is failing according to you? It would help me later when I'll be ankle to check

**Comment by abhimanyu003** [comment]
Please check this one. https://github.com/abhimanyu003/sttr/actions/runs/9260275381/job/25519173373?pr=44
Also you can run `go test -v ./...` locally. nothing major only few minor are failing.

**Comment by ccoVeille** [comment]
pushed back

**Comment by ccoVeille** [comment]
Im still unsure why the workflow is not triggered automatically

https://github.com/abhimanyu003/sttr/actions/runs/9289818782/usage

image

Does it mean you have to manually launch them ? if so, why is this set up like this?

**Comment by abhimanyu003** [comment]
Hello @ccoVeille 
Thanks for your updates.
Github action didn't get triggered automatically because there is a setting that was enabled. Because of that, I have to approve your flow first. 


> Require approval for first-time contributors
> Only first-time contributors will require approval to run workflows.

I have disabled it now. will review your PR and let you know :)

**Comment by abhimanyu003** [comment]
This is looking quite good to me. 👍 thanks so much 🙇‍♂️

**Comment by ccoVeille** [comment]
> Hello @ccoVeille 
> Thanks for your updates.
> Github action didn't get triggered automatically because there is a setting that was enabled. Because of that, I have to approve your flow first. 
> 
> 
> > Require approval for first-time contributors
> > Only first-time contributors will require approval to run workflows.
> 
> I have disabled it now. will review your PR and let you know :) 

Strange setting I have never faced until now. I learned something today 🤣

File: README.md [hunk 1]
```diff
@@ -245,14 +246,14 @@ echo "Hello World" | sttr base64-encode | sttr md5
...
 - [x] **zeropad** - Pad a number with zeros
-- [x] **and adding more....**
+- [x] **and adding more...**
 
 # Featured On
 
-These are the few locations where `sttr` was highlighted, many thanks to all of you. 
-Please feel free to add any blogs/videos you may have made that discuss `sttr` to the list.
+These are the few places where `sttr` got featured, many thanks to everyone.
+If you have talked about `sttr` in your blog/video feel to add that to the list.
```
Comment (ccoVeille) [inline README.md:254]: I'm doubtful about what happened here.

I didn't change these lines on purpose.

It looks like broken conflict resolution.

I'm sorry @abhimanyu003

File: README.md [hunk 2]
```diff
@@ -245,14 +246,14 @@ echo "Hello World" | sttr base64-encode | sttr md5
...
 
-* [Youtube: The Giants of Open Source - DevOps Paradox](https://youtu.be/4nFRKbY_HVE?t=2529?ref=abhimanyu003/sttr)
+* [YouTube: The Giants of Open Source - DevOps Paradox](https://youtu.be/4nFRKbY_HVE?t=2529?ref=abhimanyu003/sttr)
 * [Terminal Trove - Tool of the Week](https://terminaltrove.com/sttr/?ref=abhimanyu003/sttr)
 * [nixCraft](https://www.cyberciti.biz/open-source/sttr-awesome-linux-unix-command-tool-for-transformation-string/?ref=abhimanyu003/sttr)
```
Comment (ccoVeille) [inline README.md:258]: Same with these 2 lines that got added

File: processors/html.go [hunk 3]
```diff
@@ -36,6 +37,7 @@ func (p HTMLEncode) FilterValue() string {
 	return p.Title()
 }
 
+// HTMLEncode unescapes HTML to string
```
Comment (ccoVeille) [inline processors/html.go:40]: HTMLDecode here

Comment (abhimanyu003) [reply processors/html.go:40]: I'm going to push update for this.

File: README.md (hunk 1 above)
Comment (abhimanyu003) [reply README.md:254]: Yes, it looks like that. I will fix this. No worries on this. :)

**Pull request merged**